Docs/source/running_cpp/platforms.rst

```diff
@@ -43,9 +43,14 @@ regime), the following set of parameters provided good performance:
 Running on Summit at OLCF
 -------------------------
 
+.. _running-cpp-summit-V100-GPUs:
+
+V100 GPUs
+~~~~~~~~~
```

Comment:
I think technically the next headline level is `^^^^^` but this seems to work.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
Thanks, I actually referred to some unofficial documentation that uses a nonstandard convention.  I changed it, just to avoid confusion in case someone in the future adds a subsection with the expected convention..